src/pytesmo/colormaps/load_cmap.py

```diff
@@ -1,20 +1,29 @@
+"""
+Module to load color maps from json files.
+This module is deprecated and will be removed soon.
+Please use the colorella package instead:
+Website: https://github.com/TUW-GEO/colorella
+To install colorella, run 'pip install colorella'
+"""
+
 import matplotlib.cm as cm
 import matplotlib.colors as colors
 import os
 import json
 import glob
+from pytesmo.utils import deprecated
 
-
+@deprecated
```

Comment:
This code should be in colorella, therefore we dont need it anymore in pytesmo and can delete it in future